packages/vaults/src/transaction/member/member_update_name.ts

```diff
@@ -9,9 +9,20 @@ import {TransactionMapperAbstract} from "../transaction_mapper";
 import {TransactionRequest} from "../transaction_request";
 import {RequestMapperAbstract} from "../request_mapper";
 
-
+/**
+ * Interface for a transaction that updates name of an existing member.
+ * The memberId is the principal of the user with the default subaccount.
+ * This transaction can only be requested or approved only by users with the admin role.
+ * This transaction can be executed in a batch
```

Comment:
This transaction can only be requested or approved by users with the admin role.
This transaction can be executed in a batch, which means that all transactions marked with a single batch_id are executed or rejected together.